third_party/2and3/xxhash.pyi

```diff
@@ -0,0 +1,29 @@
+from _typeshed import ReadableBuffer
+from hashlib import _Hash
+
+VERSION: str
+XXHASH_VERSION: str
+
+class _IntDigestHash(_Hash):
+    def intdigest(self) -> int: ...
+
+# python-xxhash v2.0.0 does not support the string or usedforsecurity kwargs
+def xxh32(__string: ReadableBuffer = ...) -> _IntDigestHash: ...
+def xxh64(__string: ReadableBuffer = ...) -> _IntDigestHash: ...
```

Comment:
You're missing `xxh128`, an alias for `xxh3_128`.

Also, I think it's possible to specify the aliases directly in the stub, instead of duplicating the signature.